File: examples/module-10-exercises/Customer.java
public record Customer(String id, String fullName, String status) {
    // correlation note: lab-request-001

    public static void main(String[] args) {
        Customer amina = new Customer("CUS-1001", "Amina Khan", "ACTIVE");
        Customer ravi = new Customer("CUS-1002", "Ravi Singh", "PROSPECT");

        System.out.println(amina);
        System.out.println(ravi);
    }
}

File: examples/lab33-crm/crm-ui/src/types/customer.ts
export type CustomerStatus = 'PROSPECT' | 'ACTIVE' | 'CLOSED'

export interface Customer {
  customerId: string
  fullName: string
  email: string
  status: CustomerStatus
}

// export type CustomerDraft = Omit<Customer, "customerId">;

export interface CustomerDraft {
  fullName: string
  email: string
  status: CustomerStatus
}
